src/libraries/Native/Unix/System.Globalization.Native/pal_collation.c

```diff
@@ -110,10 +110,10 @@ This rule was taken from http://www.unicode.org/reports/tr35/tr35-collation.html
 */
 static int NeedsEscape(UChar character)
 {
-    return ((0x21 <= character && character <= 0x2f)
-        || (0x3a <= character && character <= 0x40)
-        || (0x5b <= character && character <= 0x60)
-        || (0x7b <= character && character <= 0x7e));
+    return 0x21 <= character && character <= 0x2f
```

Comment:
The original code used braces to make the priorities explicit. I believe some compilers or linters will even issue warning that one should use braces in more complex expressions for clarity.

Reply:
Ohh ok. I will get this reverted based on your suggestion!